build/ci.go

```diff
@@ -355,6 +356,9 @@ func doTest(cmdline []string) {
 	gotest.Args = append(gotest.Args, "-p", "1")
 	if *coverage {
 		gotest.Args = append(gotest.Args, "-covermode=atomic", "-cover", "-coverprofile=coverage.out")
+		if *coverpkg != "" {
```

Comment:
Rather than update ci.go, I think we can just use `go test ` directly, this should help reduce conflicts in ci.go.